geode-docs/tools_modules/gfsh/command-pages/query.html.md.erb

```diff
@@ -21,7 +21,11 @@ limitations under the License.
 
 Run queries against Geode regions.
 
-Run the specified OQL query as a single quoted string and displays results in pages allows to move between pages. If limit is not set in the query, then a default limit of 1000 (derived from GFSH environment variable APP\_FETCH\_SIZE) will be applied. Page size is derived from GFSH environment variable APP\_COLLECTION\_LIMIT (default value=20).
+If a limit restricting the result size is not set in the query,
+then a default limit of the gfsh environment variable `APP_FETCH_SIZE`
```

Comment:
Would it be much work to add a link to that section here in that case?  I just suspect if I was a user I would have no idea what "the gfsh environment variable `APP_FETCH_SIZE`" means.